ocsci/testlib.py

```diff
@@ -1,6 +1,10 @@
 from ocsci.pytest_customization.marks import *  # noqa: F403
+from utility import environment_check as ec
 
 
+@pytest.mark.usefixtures(  # noqa: F405
+    ec.environment_checker.__name__,
```

Comment:
You should only have to use the string 'environment_checker' here. Loading this fixture into ``tests/conftest.py`` makes is automatically accessible.

Also, why wouldn't we want to define this as an ``autouse`` fixture and just have it clean up after each test class has finished executing?

Reply:
I agree, the fixture usage is not optimal and what you are suggesting it a canonical pytest way. But since this is not fully followed in code which is already merged, I would not block this PR on this right now. It would be better to agree on change how to use fixtures, change it in all places and then start enforcing it during reviews.

Reply:
This is covered in https://github.com/red-hat-storage/ocs-ci/issues/200